docs/getting_started.md

```diff
@@ -86,8 +86,8 @@ Optional arguments:
 - `AVG_TYPE`: Items to average the test clips. If set to `prob`, it will apply softmax before averaging the clip scores. Otherwise, it will directly average the clip scores.
 - `JOB_LAUNCHER`: Items for distributed job initialization launcher. Allowed choices are `none`, `pytorch`, `slurm`, `mpi`. Especially, if set to none, it will test in a non-distributed mode.
 - `LOCAL_RANK`: ID for local rank. If not specified, it will be set to 0.
-- `--onnx`: If specified, recognition results will be generated by onnx model and `CHECKPOINT_FILE` should be onnx model file path. Onnx model files are generated by `/tools/pytorch2onnx.py`. For now, only support single-gpu mode and fixed input shape. Please note that dataset and onnx model should share the same batch size.
-- `--tensorrt`: If specified, recognition results will be generated by TensorRT engine and `CHECKPOINT_FILE` should be TensorRT engine file path. TensorRT engines files are generated by exported onnx models and TensorRT official convertion tools. For now, only support single-gpu mode and fixed input shape. Please note that dataset and TensorRT engine should share the same batch size.
+- `--onnx`: If specified, recognition results will be generated by onnx model and `CHECKPOINT_FILE` should be onnx model file path. Onnx model files are generated by `/tools/pytorch2onnx.py`. For now, multi-gpu mode and dynamic input shape mode are not supported. Please note that the output tensors of dataset and the input tensors of onnx model should share the same shape. And it is recommended to remove all test augmentation pipelines(`ThreeCrop`, `TenCrop`, `twice_sample`, etc.)
```

Comment:
And it is recommended to remove all data augmentation methods in the `test_pipeline` (`ThreeCrop`, `TenCrop`, `twice_sample`, etc.)

Reply:
Since `test time augmentation` is a widely used term, how about "test time augmentation methods in `test_pipeline`"

Reply:
testing augmentation?

Reply:
test-time augmentation?

Reply:
You're right